.pre-commit-config.yaml

```diff
@@ -43,7 +43,7 @@ repos:
     hooks:
       - id: interrogate
         args: [-vv, --config=pyproject.toml]
-        exclude: ^(docs|setup.py)|test/|__init__.py
+        pass_filenames: false
```

Comment:
documentation here: https://pre-commit.com/#hooks-pass_filenames

Reply:
This change makes `interrogate` basically behave like if you would call it from the command line.

And the whole idea behind using `interrogate` was, put simply: "Don't add new code if it isn't documented!"
Basically a test coverage for docstrings.

So at some point (close to v1 release) we can use [pydocstyle](https://github.com/PyCQA/pydocstyle) and [darglint](https://github.com/terrencepreilly/darglint) to have a good and in sync API documentation.